ingredients/DRP+fakes.yaml

```diff
@@ -1,131 +0,0 @@
-  consolidateSourceTable:
-    class: lsst.pipe.tasks.postprocess.ConsolidateSourceTableTask
-    config:
-      connections.catalogType: "fakes_"
-  consolidateVisitSummary:
-    class: lsst.pipe.tasks.postprocess.ConsolidateVisitSummaryTask
-    config:
-      connections.calexpType: "fakes_"
-  makeCcdVisitTable:
-    class: lsst.pipe.tasks.postprocess.MakeCcdVisitTableTask
-    config:
-      connections.calexpType: "fakes_"
-  makeVisitTable:
-    class: lsst.pipe.tasks.postprocess.MakeVisitTableTask
-    config:
-      connections.calexpType: "fakes_"
-  assembleCoadd:
-    class: lsst.pipe.tasks.assembleCoadd.CompareWarpAssembleCoaddTask
-    config:
-      connections.inputCoaddName: "fakes_deep"
-      connections.outputCoaddName: "fakes_deep"
-  makeWarp:
-    class: lsst.pipe.tasks.makeCoaddTempExp.MakeWarpTask
-    config:
-      connections.photoCalibName: "fgcm"
-      useGlobalExternalPhotoCalib: True
-      connections.coaddName: "fakes_deep"
-      connections.calexpType: "fakes_"
-  detection:
-    class: lsst.pipe.tasks.multiBand.DetectCoaddSourcesTask
-    config:
-      connections.inputCoaddName: "fakes_deep"
-      connections.outputCoaddName: "fakes_deep"
-  mergeDetections:
-    class: lsst.pipe.tasks.mergeDetections.MergeDetectionsTask
-    config:
-      connections.inputCoaddName: "fakes_deep"
-      connections.outputCoaddName: "fakes_deep"
-  deblend:
-    class: lsst.pipe.tasks.deblendCoaddSourcesPipeline.DeblendCoaddSourcesMultiTask
-    config:
-      connections.inputCoaddName: "fakes_deep"
-      connections.outputCoaddName: "fakes_deep"
-  measure:
-    class: lsst.pipe.tasks.multiBand.MeasureMergedCoaddSourcesTask
-    config:
-      connections.inputCoaddName: "fakes_deep"
-      connections.outputCoaddName: "fakes_deep"
-  mergeMeasurements:
-    class: lsst.pipe.tasks.mergeMeasurements.MergeMeasurementsTask
-    config:
-      connections.inputCoaddName: "fakes_deep"
-      connections.outputCoaddName: "fakes_deep"
-  writeObjectTable:
-    class: lsst.pipe.tasks.postprocess.WriteObjectTableTask
-    config:
-      connections.coaddName: "fakes_deep"
-  transformObjectTable:
-    class: lsst.pipe.tasks.postprocess.TransformObjectCatalogTask
-    config:
-      connections.coaddName: "fakes_deep"
-  forcedPhotCcd:
-    class: lsst.meas.base.forcedPhotCcd.ForcedPhotCcdTask
-    config:
-      connections.inputCoaddName: "fakes_deep"
-  forcedPhotCoadd:
-    class: lsst.meas.base.forcedPhotCoadd.ForcedPhotCoaddTask
-    config:
-      connections.inputCoaddName: "fakes_deep"
-      connections.outputCoaddName: "fakes_deep"
-  healSparsePropertyMaps:
-    class: lsst.pipe.tasks.healSparseMapping.HealSparsePropertyMapTask
-    config:
-      connections.coaddName: "fakes_deep"
-      connections.calexpType: "fakes_"
-  consolidateHealSparsePropertyMaps:
-    class: lsst.pipe.tasks.healSparseMapping.ConsolidateHealSparsePropertyMapTask
-    config:
-      connections.coaddName: "fakes_deep"
```

Comment:
I'm pretty sure it's already broken, at least because it doesn't have the `processCcdWithFakes` -> `singleFrameWithFakes` label rename that HSC/RC2+fakes has; I took that as an indication that there are probably other problems, but if you want me to make that change and include it while hoping for the best, I can.